Reference/Searching/Examine/overview-explanation.md

```diff
@@ -102,8 +102,8 @@ The easiest way to check that your config is correct is to first go and look in
 Using Luke, you can open up your index and see its content. It should give you statistics on the elements, number of documents indexed etc. If however you don't see indexed items then it's likely you have either made a mistake in your indexSet configuration or it has not indexed yet. When you are finished with Luke it's important to close the index to get Luke to release its lock on the files.
 
 Luke can also be used to run raw Lucene searches against your index. This can be helpful during development or to debug your queries. Later on in this document you will start to learn how these raw queries fit together.
-Setting up your Search Provider
 
+## Setting up your Search Provider
```

Comment:
A bit further up on the same page there is a "headline" called "Setting up your index provider" it is also not a h2 (##), not sure why this one would be then? At least we should keep it consistent and change the other one as well if this one is changed 😉

Reply:
Updated the mentioned headline to match remaining ones :)